index.js

```diff
@@ -247,8 +284,8 @@ class Tab extends EventEmitter {
             position -= this.tabGroup.tabContainer.childElementCount;
         }
 
-        if (position === 0) {
-            position = 1;
+        if (position >= 0) {
+            position++;
```

Comment:
...and this one too?

Reply:
Same as above, but this check will not be needed at all with the leftmost position being 0.